docu.js

```diff
@@ -1,9 +1,11 @@
-import { cardCreation } from "./src/content/card.js";
+import * as main from "./main.js";
+
+/*import { cardCreation } from "./src/content/card.js";
```

Comment:
A comment for your comment: This comment is just commented out code, so I would recommend removing it :)

Reply:
i know! clean up this week! ;) Thx